README.md

```diff
@@ -1,22 +1,49 @@
 # Frequenz Dispatch Client Library
 
 [![Build Status](https://github.com/frequenz-floss/frequenz-client-dispatch-python/actions/workflows/ci.yaml/badge.svg)](https://github.com/frequenz-floss/frequenz-client-dispatch-python/actions/workflows/ci.yaml)
-[![PyPI Package](https://img.shields.io/pypi/v/frequenz-client-dispatch)](https://pypi.org/project/frequenz-client-dispatch/)
-[![Docs](https://img.shields.io/badge/docs-latest-informational)](https://frequenz-floss.github.io/frequenz-client-dispatch-python/)
+[![PyPI Version](https://img.shields.io/pypi/v/frequenz-client-dispatch)](https://pypi.org/project/frequenz-client-dispatch/)
+[![Documentation](https://img.shields.io/badge/docs-latest-brightgreen)](https://frequenz-floss.github.io/frequenz-client-dispatch-python/)
 
-## Introduction
+## 🚀 Introduction
 
-Dispatch API client for Python
+Welcome to the **Frequenz Dispatch Client Library**—your go-to Python client for low-level interactions with the Frequenz Dispatch API!
 
-## Supported Platforms
+If you're a developer who needs direct access to the [Dispatch API](https://github.com/frequenz-floss/frequenz-dispatch-api) without the abstraction layers of the high-level client, you're in the right place. This library serves as the foundation for our more feature-rich and user-friendly [high-level client](https://github.com/frequenz-floss/frequenz-dispatch-python).
 
-The following platforms are officially supported (tested):
+## 📦 Installation
+
+Install the library via pip:
+
+```bash
+pip install frequenz-client-dispatch
+```
+
+## 🛠️ Usage
+
+Here's a quick example to get you started:
+
+```python
+from frequenz.client.dispatch import Client
+
+# Initialize the client
```

Comment:
I'd suggest to wrap all the code within an async function. I know this will make the example much more verbose but otherwise the user might try to instantiate the client outside the asynchronous event loop.

For instance:
```python
import asyncio
from frequenz.client.dispatch import Client

async def list_dispatches() -> None:
    # Initialize the client
    client = Client(key="your_api_key", server_url="grpc://fz-0004.frequenz.io")

    # List all dispatches for a specific microgrid
    async for page in client.list(microgrid_id=1):
        for dispatch in page:
            print(dispatch)

asyncio.run(list_dispatches())
```